ExtIO_sddc/ExtIO_sddc.cpp

```diff
@@ -404,16 +410,21 @@ extern "C"
 long EXTIO_API GetHWLO(void)
 {
     EnterFunction();
-
-	return (long)(glLOfreq & 0xFFFFFFFF);
+	if (RadioHandler.GetFine_LO())
```

Comment:
why not make glLocoor == 0 in the other case, then we can normalize the code here just return glLOfreq + glLOcorr

Reply:
Ok done